Intro_markdown.Rmd

```diff
@@ -156,7 +156,7 @@ getwd()
 If you want to change the working directory to a specific repository/folder, you can use the `setwd()` command as follows:
```

Comment:
Add "in this case the IntroRTraining one" after the comma?

Reply:
done